nova/policy.py

```diff
@@ -235,6 +235,22 @@ def __call__(self, target, creds, enforcer):
         return creds['is_admin'] == self.expected
 
 
+@policy.register('target_has_no_project_id')
+class TargetHasNoProjectIdCheck(policy.Check):
+
+    def __init__(self, kind, match):
+
+        self.expected = (match.lower() == 'true')
+
+        super(TargetHasNoProjectIdCheck, self).__init__(kind,
```

Comment:
aren't we on python3 yet and can use plain `super()`?

Reply:
probably? the copied code right above doesn't do that though and I usually opt for consistency inside a file.

Reply:
fair. although it makes the line break and the code a bit harder to read here.